keanu-python/tests/test_vertex.py

```diff
@@ -278,3 +299,12 @@ def test_you_can_observe(ctor, args, expected_type, value, assert_vertex_value_e
     vertex.observe(value)
     assert vertex.is_observed()
     assert_vertex_value_equals(vertex, expected_type, value)
+
+
+@pytest.mark.parametrize("ctor, args, expected_type", [(Gaussian, (0., 1.), float), (UniformInt, (0, 10), int),
+                                                       (Bernoulli, (0.5,), bool)])
+@pytest.mark.parametrize("value, assert_vertex_value_equals", [(4, assert_vertex_value_equals_scalar),
+                                                               (5., assert_vertex_value_equals_scalar),
+                                                               (True, assert_vertex_value_equals_scalar)])
+def test_you_can_observe_scalar(ctor, args, expected_type, value, assert_vertex_value_equals):
+    test_you_can_observe(ctor, args, expected_type, value, assert_vertex_value_equals)
```

Comment:
Also you can call `assert_vertex_value_equals_scalar` directly here instead of passing it as a parameter

Reply:
👍